test/onigumo_test.exs

```diff
@@ -76,6 +79,6 @@ defmodule OnigumoTest do
   end
 
   defp body(url) do
-    "Body from: #{url}"
+    "Body from: #{url}\n"
```

Comment:
Oh. This should have already been there. I made a mistake when merging #43 and accidentally removed the newline. Fixed in https://github.com/Glutexo/onigumo/commit/35cce947a85f99114abad41b4314c08a61f42068 directly pushed to [_elixir_](https://github.com/Glutexo/onigumo/tree/elixir).